docs/content/docs/concepts/overview.md

```diff
@@ -88,7 +88,7 @@ The examples are maintained as part of the operator repo and can be found [here]
 ## Known Issues & Limitations
 
 ### JobManager High-availability
```

Comment:
TBH I don't know why this was in the "Known Issues & Limitations" section? Maybe we can move it somewhere else?